cartoframes/data/observatory/entity.py

```diff
@@ -14,68 +12,44 @@ class CatalogEntity(ABC):
     id_field = None
     entity_repo = None
 
-    @classmethod
-    @abstractmethod
-    def _get_single_entity_class(cls):
-        raise NotImplementedError
+    def __init__(self, data):
+        self.data = data
 
-    @classmethod
-    @abstractmethod
-    def _get_entities_list_class(cls):
-        raise NotImplementedError
+    @property
+    def id(self):
+        return self.data[self.id_field]
 
     @classmethod
-    def get_by_id(cls, id_):
+    def get(cls, id_):
         return cls.entity_repo.get_by_id(id_)
 
+    @classmethod
+    def get_all(cls):
+        return cls.entity_repo.get_all()
+
+    def to_series(self):
+        return pd.Series(self.data)
+
     def __eq__(self, other):
-        return self.equals(other)
+        return self.data == other.data
 
     def __ne__(self, other):
         return not self == other
 
+    def __str__(self):
+        return '{classname}({data})'.format(classname=self.__class__.__name__, data=self.data.__str__())
```

Comment:
We're returning just the data structure. I think it's confusing because the user will think the returned structure is a list of dictionaries when it's not. The returned structure is an object.

We talked about adding the data + the name of the class:
```python
Country({'id': 'es'})
```
However, with objects with too many fields, it will be difficult to read. To fix it, I propose:

1. Define `__repr__` to <classname>(<object_id>). Example: Country('esp')
2. Add `to_dict` method to return the data as a dictionary.

Reply:
Ok, so I have changed the __repr__ of the entities so it returns only the id. 

Also, I have overridden those methods in the CatalogList so:
   - __str__ returns the full items with the class name (__str__ of lists by default calls the __repr__ of its items)
   - __repr__ returns the the id of the items with the class name

I added a to_dict method as well